tools/docker/builder/ubuntu/bionic/Dockerfile

```diff
@@ -26,5 +26,34 @@ RUN apt-get update && apt-get install -y \
     python3-yaml \
     uuid-runtime \
     valgrind \
-    vim \
-    && rm -rf /var/lib/apt/lists/*
+    vim
+# AMBRIORIX dependencies
+RUN \
+    apt-get install -y \
+    bison \
+    flex \
+    libevent-dev \
+    libyajl-dev \
+    repo && \
+    echo "deb http://cz.archive.ubuntu.com/ubuntu eoan main universe" | tee -a  /etc/apt/sources.list && \
+    apt-get update && \
+    apt-get install -y \
+    liburiparser-dev && \
```

Comment:
Default repos for Ubuntu 18.04 only have `liburiparser-dev 8.4` available. AMBIORIX uses some symbols which are available since 9.0 version. That's why we are using repo from Ubuntu 19.10.

Reply:
Added commentary explaining why do we get `liburiparser-dev` from Ubuntu Eon repos.